code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -128,15 +128,19 @@
 		)
 
 /datum/species/unathi/ashwalker/on_species_gain(mob/living/carbon/human/H)
+	..()
```

Comment:
Hey, please make sure the firebreath from #21981 doesn't break cause of this - it did wacky stuff in testing when I added it, which is why I may have not included the parent call... wrongly... no excuses.